requirements.txt

```diff
@@ -10,4 +10,5 @@ matplotlib<3.10.0
 scipy
 protobuf
 mct-quantizers==1.5.2
-pydantic<2.0
\ No newline at end of file
+pydantic<2.0
+sony-custom-layers==0.4.0
```

Comment:
didn't we say it needs to be >=0.4.0?

Reply:
I don't really care, I thought @Idan-BenAmi said that we should set >=.
I also think it is better because that way we won't need to update the requirement every CL release